utils/common.sh

```diff
@@ -194,3 +194,30 @@ EOF
 }
 
 
+##############################################################################
+# Saves a workload backup in the snappy server
+# Arguments:
+#   File terminations list
+#   File list
+#   Workload name
+##############################################################################
+snappy_backup(){
+  log "Backing up stuff in snappy server"
+  local files
+  mkdir -p files_list
+  source snappy-move-results/common.sh
+  for term in ${1}; do
+    files=$(find . -name "*.${term}")
+    cp ${files} files_list
+  done
+  for f in ${2}; do
+    cp ${f} files_list
+  done
+  tar czf snappy_files.tar.gz files_list metadata.json
+  local snappy_path="${SNAPPY_USER_FOLDER}/${runid}${platform}-${cluster_version}/${workload}/${folder_date_time}/"
```

Comment:
before this need to export wokload= ${3}

Reply:
fixed!